src/main/java/com/example/App.java

```diff
@@ -0,0 +1,22 @@
+package com.example;
+
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.CommandLineRunner;
+import org.springframework.boot.SpringApplication;
+import org.springframework.boot.autoconfigure.SpringBootApplication;
+
+@SpringBootApplication
+public class App implements CommandLineRunner {
+
+    @Autowired
+    private TicketService ticketService;
+
+    public static void main(String[] args) {
+        SpringApplication.run(App.class, args);
+    }
+
+    @Override
+    public void run(String... args) throws Exception {
+        ticketService.run();
+    }
```

Comment:
Here you can only use the main method. All logic should be used from the controller. That is, you add some request to the controller, the controller accesses the service and works with it